js/web/README.md

```diff
@@ -0,0 +1,153 @@
+# ONNX Runtime Web
+
+ONNX Runtime Web is a Javascript library for running ONNX models on browsers and on Node.js.
+
+ONNX Runtime Web has adopted WebAssembly and WebGL technologies for providing an optimized ONNX model inference runtime for both CPUs and GPUs.
+
+### Why ONNX models
+
+The [Open Neural Network Exchange](http://onnx.ai/) (ONNX) is an open standard for representing machine learning models. The biggest advantage of ONNX is that it allows interoperability across different open source AI frameworks, which itself offers more flexibility for AI frameworks adoption. See [Getting ONNX Models](#Getting-ONNX-models).
+
+### Why ONNX Runtime Web
+
+With ONNX Runtime Web, web developers can score pre-trained ONNX models directly on browsers with various benefits of reducing server-client communication and protecting user privacy, as well as offering install-free and cross-platform in-browser ML experience.
+
+ONNX Runtime Web can run on both CPU and GPU. For running on CPU, [WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly) is adopted to execute the model at near-native speed. Furthermore, ONNX Runtime Web utilizes [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) to provide a "multi-threaded" environment to parallelize data processing. Empirical evaluation shows very promising performance gains on CPU by taking full advantage of WebAssembly and Web Workers. For running on GPUs, a popular standard for accessing GPU capabilities - WebGL is adopted. ONNX Runtime Web has further adopted several novel optimization techniques for reducing data transfer between CPU and GPU, as well as some techniques to reduce GPU processing cycles to further push the performance to the maximum.
+
+See [Compatibility](#Compatibility) and [Operators Supported](#Operators) for a list of platforms and operators ONNX Runtime Web currently supports.
+
+## Getting Started
+
+There are multiple ways to use ONNX Runtime Web in a project:
+
+### Using `<script>` tag
+
+This is the most straightforward way to use ONNX Runtime Web. The following HTML example shows how to use it:
+
+```html
+<html>
+  <head> </head>
+
+  <body>
+    <!-- Load ONNX Runtime Web -->
+    <script src="https://cdn.jsdelivr.net/npm/onnxruntime-web/dist/ort.min.js"></script>
```

Comment:
Did you change ort.min.js to ort-web.min.js in other commit?

Reply:
not in examples. it's in onnxruntime-web's package.json

Reply:
actually, ort.min.js = ort-common.min.js + ort-web.min.js